control/tests/test_deliver_cielo.py

```diff
@@ -146,13 +146,13 @@ def test_transcript_flow(self):
                     v=CIELO24_API_VERSION,
                     job_id='000-111-222',
                     target_language='TARGET_LANG',
-                    callback_url=build_url(
+                    callback_url='{}?job_id={}&iwp_name={}&lang_code={}&org={}&video_id={}'.format(
```

Comment:
I think we should use format variables here like `{cielo_callback_base_url}?job_id={job_id}&iwp_name={iwp_name}&lang_code={lang_code}&org={org}&video_id={video_id}`

Reply:
This will make the line longer and pylint will complain.